proto/Cargo.toml

```diff
@@ -14,7 +14,9 @@ hex = "0.4"
 anyhow = "1.0"
 subtle-encoding = "0.5"
 bech32 = "0.8"
+ibc-proto = { git = "https://github.com/penumbra-zone/ibc-rs" }
```

Comment:
What's the motivation for having our own fork here?

Reply:
Originally it's so that we could derive the `prost::Message` trait, which is required for the `Oneof` in the IBCAction proto, but in the latest ibc-rs version that appears to already exist. We can just use that, I don't see a reason to use our own fork at this point